portfolio/src/main/webapp/js/scripts.js

```diff
@@ -1,3 +1,7 @@
+//global variables for map
+let geocoder;
```

Comment:
does geocoder need to be global?

Reply:
Good point, it doesn't need to be. I changed it so that only map is a global variable. Thanks!